app/controllers/orders_controller.rb

```diff
@@ -21,35 +22,57 @@ def update
     render :show
   end
 
+  def index
+    @merchant = User.find(params[:user_id])
```

Comment:
I see this line repeated several times in this controller so it might be a good place for a before_action or something along those lines